stdlib/Makefile

```diff
@@ -0,0 +1,34 @@
+default: get
+
+PKG3_VERSION := f4139648bfb4cdde34e50947a73f71c7ee8d29a5
+PKG3_URL := https://github.com/vchuravy/Pkg3.jl
+
+Pkg3:
+	git clone $(PKG3_URL) $@
+Pkg3/$(PKG3_VERSION).commit: Pkg3
+	git -C $< checkout $(PKG3_VERSION)
```

Comment:
I think you need to use the tools defined in `deps/tools/git-external.mk`.  Basically we define both a git url and a tarball url, so that if the user wants to use `git` they can, but if they want to just use `curl` and `tar` they can do that as well.

Reply:
Thanks I will do that!